data/levels/revenge_in_redmond/intro.txt

```diff
@@ -0,0 +1,32 @@
+(supertux-text			
+  (background	"retro/intro.jpg")
+  (music	"music/retro/theme_old.music")
+  (text "-A Blast From The Past!
+ Or was it a dream?
+
+	Tux and Gown were having a nice picnic
+	when suddenly Gown is beamed away!!!
+	
+	Wait a second... Gown?
+	Who is this penguin that looks
+	an awful lot like his friend Penny.
+	
+	Also a helicopter?
+	Since when do helicopters casually fly
+	over Antarctica and abduct penguins?
+	
+	Even though Tux was visibly confused
+	about the events that just transpired,
+	he could not just stand there
+	doing nothing, therefore, Tux
+	chased after to rescue this

```

Comment:
I think you need a noun here (but I'm not sure):
`chased after <sth>`

Reply:
I personally think you don't need it here. It also sounds worse honestly flow-wise. Maybe "chased after it" could work here if need be.